app/models/concerns/user_multifactor_methods.rb

```diff
@@ -26,7 +26,7 @@ def no_mfa_devices?
   end
 
   def mfa_gem_signin_authorized?(otp)
-    return true unless strong_mfa_level? || webauthn_credentials.present?
+    return true unless strong_mfa_level? || webauthn_enabled?
```

Comment:
```suggestion
    return true unless strong_mfa_level?
```
This can be removed once the [rake task](https://github.com/rubygems/rubygems.org/blob/master/lib/tasks/one_time.rake#L7) to migrate all webauthn only users is run.